pyproject.toml

```diff
@@ -26,6 +26,12 @@ dependencies = [
     "numpy"
 ]
 
+[tool.black]
+line-length = 88
+target-version = ['py37', 'py38', 'py39', 'py310']
```

Comment:
Add py311?

Reply:
Not in pyproject.toml?
```
classifiers = [
    "Development Status :: 5 - Production/Stable",
    "Intended Audience :: Science/Research",
    "License :: OSI Approved :: BSD License",
    "Programming Language :: Python",
    "Programming Language :: Python :: 3.7",
    "Programming Language :: Python :: 3.8",
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3.10",
    "Programming Language :: C++"
]
```

Reply:
Also we do not -- yet -- build against 3.11. I'd suggest adding this when it's the time.